src/routes/campaigns/campaignId/_get/index.spec.ts

```diff
@@ -129,6 +159,28 @@ describe("GET /campaigns/:campaignId", () => {
       .set("Authorization", "Bearer admin");
     expect(response.body).toHaveProperty("preselectionFormId", 20);
   });
+  it("Should return plan if exist", async () => {
+    const response = await request(app)
+      .get("/campaigns/2")
+      .set("Authorization", "Bearer admin");
+    expect(response.body).toHaveProperty("plan");
+    expect(response.body.plan).toMatchObject({
+      id: 2,
+      name: "Form CP2",
+    });
+  });
+  it("Should not return plan if campaign plan does not exist", async () => {
+    const response = await request(app)
+      .get("/campaigns/1")
+      .set("Authorization", "Bearer admin");
+    expect(response.body).toEqual({
+      id: 1,
+      title: "This is the title",
+      type: "functional",
+      typeDescription: "functional description",
+      plan: undefined,
+    });
+  });
   it("Should not return campaign preselectionFormId if campaign form does not exist", async () => {
```

Comment:
inserirei anche un test che verifica se una cp ha un plan vecchio (che non esiste più, che è stato rimosso) e non deve andare in errore e non ritornare nessun plan nel body

Reply:
dovrebbe essere safe perchè è gestito da db

Reply:
on delete